.github/ISSUE_TEMPLATES/2-feature-request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Request a new feature for the Props Bot.
+labels: ['[Type] Feature Request']
+assignees:
+  - desrosj
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to submit this feature request! Keep in mind, there's no promise that all feature requests will be accepted.
+        This action has a specific purpose, make tracking all contributions the WordPress project on Github easier to track collect in an automated way.
```

Comment:
`This action has a specific purpose, make tracking all contributions the WordPress project on Github easier to track collect in an automated way.` >> `This action has a specific purpose, to make tracking all contributions to the WordPress project on GitHub easier to track and collect in an automated way.`

Reply:
Much better. I did remove the second "track" in the sentence though. It felt repetitive.